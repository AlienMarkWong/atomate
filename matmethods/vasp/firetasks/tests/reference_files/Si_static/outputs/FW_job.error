INFO:root:Starting job no. 1 (Vasp Job) attempt no. 1. Errors thus far = 0.
ModuleCmd_Load.c(200):ERROR:105: Unable to locate a modulefile for 'usg-default-modules'
ModuleCmd_Load.c(200):ERROR:105: Unable to locate a modulefile for 'python/2.7.3'
ModuleCmd_Load.c(200):ERROR:105: Unable to locate a modulefile for 'git'
ModuleCmd_Load.c(200):ERROR:105: Unable to locate a modulefile for 'python/2.7.3'
ModuleCmd_Load.c(200):ERROR:105: Unable to locate a modulefile for 'python/2.7.3'
ModuleCmd_Load.c(200):ERROR:105: Unable to locate a modulefile for 'python/2.7.3'
ModuleCmd_Load.c(200):ERROR:105: Unable to locate a modulefile for 'python/2.7.3'
ModuleCmd_Load.c(200):ERROR:105: Unable to locate a modulefile for 'python/2.7.3'
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading scstep...
DEBUG:pymatgen.io.vaspio.vasp_output:Reading eigenvalues. Projected = False
DEBUG:pymatgen.io.vaspio.vasp_output:Reading spin 1
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 1
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 2
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 3
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 4
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 5
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 6
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 7
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 8
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 9
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 10
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 11
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 12
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 13
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 14
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 15
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 16
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 17
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 18
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 19
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 20
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 21
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 22
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 23
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 24
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 25
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 26
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 27
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 28
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 29
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 30
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 31
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 32
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 33
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 34
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 35
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 36
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 37
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 38
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 39
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 40
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 41
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 42
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 43
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 44
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 45
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 46
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 47
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 48
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 49
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 50
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 51
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 52
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 53
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 54
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 55
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 56
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 57
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 58
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 59
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 60
DEBUG:pymatgen.io.vaspio.vasp_output:Reading spin -1
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 1
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 2
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 3
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 4
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 5
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 6
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 7
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 8
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 9
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 10
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 11
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 12
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 13
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 14
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 15
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 16
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 17
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 18
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 19
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 20
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 21
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 22
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 23
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 24
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 25
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 26
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 27
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 28
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 29
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 30
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 31
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 32
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 33
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 34
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 35
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 36
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 37
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 38
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 39
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 40
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 41
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 42
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 43
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 44
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 45
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 46
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 47
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 48
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 49
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 50
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 51
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 52
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 53
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 54
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 55
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 56
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 57
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 58
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 59
DEBUG:pymatgen.io.vaspio.vasp_output:Reading kpoint 60
DEBUG:pymatgen.io.vaspio.vasp_output:Finished reading eigenvalues. No. eigen = 120
DEBUG:pymatgen.io.vaspio.vasp_output:Reading dos...
INFO:root:Logging to custodian.json...
INFO:root:Run completed
INFO:rocket.launcher:Rocket finished
