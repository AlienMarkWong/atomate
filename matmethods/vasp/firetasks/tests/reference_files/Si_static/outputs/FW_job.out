Loading torque prologue
Done
mp_prod environment
2013-09-21 11:51:45,547 INFO Launching Rocket
2013-09-21 11:51:46,020 DEBUG Querying for duplicates, fw_id: 389927
2013-09-21 11:51:46,028 DEBUG Verifying for duplicates, fw_ids: 389927, 389927
2013-09-21 11:51:46,045 DEBUG FW with id: 389927 is unique!
2013-09-21 11:51:46,051 DEBUG Created/updated Launch with launch_id: 241985
2013-09-21 11:51:46,176 DEBUG Checked out FW with id: 389927
COPYING /project/projectdirs/matgen/garden/aflow_engines-mag_special/runs_2011-08-25-09-51-06/2011-5-27-17-51-13_11721_79021/relax2/INCAR INCAR
COPYING /project/projectdirs/matgen/garden/aflow_engines-mag_special/runs_2011-08-25-09-51-06/2011-5-27-17-51-13_11721_79021/relax2/KPOINTS KPOINTS
COPYING /project/projectdirs/matgen/garden/aflow_engines-mag_special/runs_2011-08-25-09-51-06/2011-5-27-17-51-13_11721_79021/relax2/POTCAR POTCAR
COPYING /project/projectdirs/matgen/garden/aflow_engines-mag_special/runs_2011-08-25-09-51-06/2011-5-27-17-51-13_11721_79021/relax2/OUTCAR OUTCAR
COPYING /project/projectdirs/matgen/garden/aflow_engines-mag_special/runs_2011-08-25-09-51-06/2011-5-27-17-51-13_11721_79021/relax2/vasprun.xml vasprun.xml
COPYING /project/projectdirs/matgen/garden/aflow_engines-mag_special/runs_2011-08-25-09-51-06/2011-5-27-17-51-13_11721_79021/relax2/OSZICAR OSZICAR
COPYING /project/projectdirs/matgen/garden/aflow_engines-mag_special/runs_2011-08-25-09-51-06/2011-5-27-17-51-13_11721_79021/relax2/CONTCAR POSCAR
running on MENDEL
2013-09-21 11:52:37,730 INFO Rocket finished

----------------------------------------------------------------
Jobs exit status code is 0
Job Si1--GGA_static_v2/6671258.cvrsvc09-ib completed Sat Sep 21 11:52:38 PDT 2013
Submitted by matcomp/matcomp using matcomp
Job Limits: neednodes=2:ppn=16,nodes=2:ppn=16,walltime=168:00:00
Job Resources used: cput=00:19:56,mem=969448kb,vmem=5453564kb,walltime=00:00:53
Nodes used: mc0803 mc0856 

Killing any leftover processes...

Job completed.
